financialaid/models.py

```diff
@@ -27,10 +27,11 @@ class TierProgram(models.Model):
     """
     The tiers for discounted pricing assigned to a program
     """
-    program = models.ForeignKey(Program, null=False)
+    program = models.ForeignKey(Program, null=False, related_name="tier_programs")
     tier = models.ForeignKey(Tier, null=False)
-    value = models.IntegerField(null=False)
+    discount_amount = models.IntegerField(null=False)
```

Comment:
Just to make a note of the reasoning: if we uses percentage off coupons, it limits the potential values we can set for adjusted price. But if we use fixed discounts, we can potentially set any price.